paper/paper.md

```diff
@@ -0,0 +1,127 @@
+contain all of the necessary cost and performance data for different technology
+classes. `osier` comes pre-loaded with a variety of technologies described in
+the National Renewable Energy Laboratory's (NREL) Annual Technology Baseline
+(ATB) dataset[@nationalrenewableenergylaboratory:2023] but users are also able
+to define their own.
+
+A set of `osier.Technology` objects, along with user-supplied demand data, can
+be tested independently with the `osier.DispatchModel`. The
+`osier.DispatchModel` is a linear programming model implemented with the `pyomo`
+library [@hart:2011]. For investment decisions and tradeoff analysis, users can
+pass their portfolio of `osier.Technology` objects, energy demand, and their
+desired objectives to the `osier.CapacityExpansion` model, the highest level
+model in `osier`. The `osier.CapacityExpansion` model is implemented with the
+multi-objective optimization framework, `pymoo` [@blank:2020].
+\autoref{fig:osier-flow} overviews the flow of data through `osier`.
+
+![The flow of data into and within `osier`
+\label{fig:osier-flow}](osier_flow.png)
+
+## Key Features
+In addition to being the first and only open-source multi-objective energy
+modelling framework, `osier` has a few key features that further distinguishes
+it from other modelling frameworks. First, since `osier.Technology` objects are
+Python objects, users can modify values and assumptions, or assign new
+attributes to the tested technologies. Second, contrary to conventional energy
+system models, `osier` has no required objectives. While users may choose from a
+variety of pre-defined objectives, they may also declare their own objectives
+based on any quantifiable metric. The requirements for a bespoke objective are: 
+
+1. The first argument must be a list of `osier.Technology` objects.
+2. The second argument must be the results from an `osier.DispatchModel`. But
+   this may be a simple placeholder with a default value of `None`.
+3. The function must return a single numerical value.
+4. The final requirement, is that all `osier.Technology` objects
+   possess the attribute being optimized.
+
+These two features acknowledge that a modeler cannot know *a priori* all
+possible objectives or parameters of interest. Allowing users to define their
+own objectives and modify technology objects (or simply build their own by
+inheriting from the `osier.Technology` class) accounts for this limitation and
+expands the potential for incorporating localized preferences. Lastly, in order
+to account for unmodeled or unmodelable objectives, `osier` extends the
+conventional MGA algorithm into N-dimensions by using a farthest-first-traversal
+in the design space.
+
+## Documentation
+
+`osier` offers robust documentation with detailed usage examples at
+[osier.readthedocs.io](https://osier.readthedocs.io).
+
+# Acknowledgements
+
+Samuel Dotson was supported by the Nuclear Regulatory Commission Fellowship
```

Comment:
Your acknowledgements should include the people who reviewed your PRs for Osier.

Reply:
Use the credit taxonomy to define who did what for this research output: https://credit.niso.org/
https://theplosblog.plos.org/2022/01/contributorship/